### DISCUSSION TITLE
404 Help

### DISCUSSION THREAD

**Discussion by HWyyds** [spencerwooo/onedrive-vercel-index#645] [Q&A]
今天访问的时候突然给我404报错
提示如下：
`"<!DOCTYPE html><html lang=\"en\"><head><meta name=\"description\" content=\"OneDrive Vercel Index\"/><link rel=\"icon\" href=\"/favicon.ico\"/><link rel=\"preconnect\" href=\"https://fonts.googleapis.com\"/><link rel=\"preconnect\" href=\"https://fonts.gstatic.com\" crossorigin=\"true\"/><link rel=\"stylesheet\" data-href=\"https://fonts.googleapis.com/css2?family=Fira+Mono&amp;family=Inter:wght@400;500;700&amp;display=swap\"/><meta name=\"viewport\" content=\"width=device-width\"/><meta charSet=\"utf-8\"/><title>500: Internal Server Error</title><meta name=\"next-head-count\" content=\"3\"/><link rel=\"preload\" href=\"/_next/static/css/a19d256284a2f006.css\" as=\"style\"/><link rel=\"stylesheet\" href=\"/_next/static/css/a19d256284a2f006.css\" data-n-g=\"\"/><noscript data-n-css=\"\"></noscript><script defer=\"\" nomodule=\"\" src=\"/_next/static/chunks/polyfills-5cd94c89d3acac5f.js\"></script><script src=\"/_next/static/chunks/webpack-391bd34453307552.js\" defer=\"\"></script><script src=\"/_next/static/chunks/framework-cda9361f9e2accd2.js\" defer=\"\"></script><script src=\"/_next/static/chunks/main-65b29fb35a55f240.js\" defer=\"\"></script><script src=\"/_next/static/chunks/pages/_app-478dd46143f52d20.js\" defer=\"\"></script><script src=\"/_next/static/chunks/pages/_error-bd8fe4fe4c21621e.js\" defer=\"\"></script><script src=\"/_next/static/ueHsuAWXEPc793m9aObWn/_buildManifest.js\" defer=\"\"></script><script src=\"/_next/static/ueHsuAWXEPc793m9aObWn/_ssgManifest.js\" defer=\"\"></script><script src=\"/_next/static/ueHsuAWXEPc793m9aObWn/_middlewareManifest.js\" defer=\"\"></script><style id=\"__jsx-2283583613\">#nprogress{pointer-events:none;}#nprogress .bar{background:rgb(156, 163, 175, 0.9);position:fixed;z-index:9999;top:0;left:0;width:100%;height:1px;}#nprogress .peg{display:block;position:absolute;right:0px;width:100px;height:100%;box-shadow:0 0 10px rgb(156, 163, 175, 0.9),0 0 5px rgb(156, 163, 175, 0.9);opacity:1;-webkit-transform:rotate(3deg) translate(0px,-4px);-ms-transform:rotate(3deg) translate(0px,-4px);-webkit-transform:rotate(3deg) translate(0px,-4px);-ms-transform:rotate(3deg) translate(0px,-4px);transform:rotate(3deg) translate(0px,-4px);}#nprogress .spinner{display:block;position:fixed;z-index:1031;top:15px;right:15px;}#nprogress .spinner-icon{width:18px;height:18px;box-sizing:border-box;border:solid 2px transparent;border-top-color:rgb(156, 163, 175, 0.9);border-left-color:rgb(156, 163, 175, 0.9);border-radius:50%;-webkit-animation:nprogresss-spinner 400ms linear infinite;-webkit-animation:nprogress-spinner 400ms linear infinite;animation:nprogress-spinner 400ms linear infinite;}.nprogress-custom-parent{overflow:hidden;position:relative;}.nprogress-custom-parent #nprogress .spinner,.nprogress-custom-parent #nprogress .bar{position:absolute;}@-webkit-keyframes nprogress-spinner{0%{-webkit-transform:rotate(0deg);}100%{-webkit-transform:rotate(360deg);}}@-webkit-keyframes nprogress-spinner{0%{-webkit-transform:rotate(0deg);-ms-transform:rotate(0deg);transform:rotate(0deg);}100%{-webkit-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg);}}@keyframes nprogress-spinner{0%{-webkit-transform:rotate(0deg);-ms-transform:rotate(0deg);transform:rotate(0deg);}100%{-webkit-transform:rotate(360deg);-ms-transform:rotate(360deg);transform:rotate(360deg);}}</style><link rel=\"stylesheet\" href=\"https://fonts.googleapis.com/css2?family=Fira+Mono&family=Inter:wght@400;500;700&display=swap\"/></head><body><div id=\"__next\" data-reactroot=\"\"><div style=\"color:#000;background:#fff;font-family:-apple-system, BlinkMacSystemFont, Roboto, &quot;Segoe UI&quot;, &quot;Fira Sans&quot;, Avenir, &quot;Helvetica Neue&quot;, &quot;Lucida Grande&quot;, sans-serif;height:100vh;text-align:center;display:flex;flex-direction:column;align-items:center;justify-content:center\"><div><style>body { margin: 0 }</style><h1 style=\"display:inline-block;border-right:1px solid rgba(0, 0, 0,.3);margin:0;margin-right:20px;padding:10px 23px 10px 0;font-size:24px;font-weight:500;vertical-align:top\">500</h1><div style=\"display:inline-block;text-align:left;line-height:49px;height:49px;vertical-align:middle\"><h2 style=\"font-size:14px;font-weight:normal;line-height:inherit;margin:0;padding:0\">Internal Server Error<!-- -->.</h2></div></div></div></div><script id=\"__NEXT_DATA__\" `type=\"application/json\">{\"props\":{\"pageProps\":{\"statusCode\":500}},\"page\":\"/_error\",\"query\":{},\"buildId\":\"ueHsuAWXEPc793m9aObWn\",\"nextExport\":true,\"isFallback\":false,\"gip\":true,\"locale\":\"en\",\"locales\":[\"en\",\"zh-CN\"],\"defaultLocale\":\"en\",\"scriptLoader\":[]}</script></body></html>"`
网站：cloud.fantanstic.top
试问下各位大佬怎么解决

**Comment by zkeq** [comment]
大概是 `redis` 用超了？

在用的是 `Upstash` 吗？

https://console.upstash.com/

**Reply by HWyyds** [reply]
对
但我看了下貌似并没有用超

**Reply by D3-3109** [reply]
删掉REDIS里边的值然后回页面重新部署一次试试？

**Comment by minterjia** [comment]
如何解决的大佬
